src/Plugin.php

```diff
@@ -130,7 +130,25 @@ public static function renderTemplate(array $template_subpathnames, array $varia
    * @return bool
    */
   public static function isLazyLoadActive() {
-    return is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    $lazy_load_is_active = static::is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
```

Comment:
As we discussed in person, I'd get rid of this line and define the default value as `FALSE`.